File: src/main/java/de/neosit/minecraft/arduinointegration/PotiUtil.java
package de.neosit.minecraft.arduinointegration;

public final class PotiUtil {
	
	public static Boolean checkForPotiInput(String input) {
		return input.startsWith("poti:");
	}
	
	public static String createPotiCommand(String input, Configuration config) {
		if (checkForPotiInput(input)) {
			String value = input.substring(input.lastIndexOf(":") + 1);
			String command = config.getCommandForArduinoValue("poti");
			command = command.replace("?", value);
			return command;
		}
		return null;
	}
}

File: Projekt/src/main/java/de/neosit/minecraft/arduinointegration/PotiUtil.kt
package de.neosit.minecraft.arduinointegration

object PotiUtil {

    fun checkForPotiInput(input: String): Boolean {
        return input.startsWith("poti:")
    }

    fun createPotiCommand(input: String, config: Configuration): String? {
        if (checkForPotiInput(input)) {
            val value = input.split(':')[1]
            var command = config.getCommandForArduinoValue("poti")
            command = command.replace("?", value)
            return command
        }
        return null
    }
}
